docs/testing.rst

```diff
@@ -79,4 +79,4 @@ that was written in it without having to create temporary files with
 :py:mod:`tempfile`.
 
 
-.. _PyHamcrest: http://pythonhosted.org//PyHamcrest/
+.. _PyHamcrest: http:///PyHamcrest.readthedocs.io/
```

Comment:
(Thanks, my script did not look at cases or multiple slashes... Something to remember for next time.)